conda/recipes/cudf/meta.yaml

```
@@ -24,7 +24,7 @@ requirements:
     - setuptools
     - numba >=0.49.0
     - dlpack
-    - pyarrow 0.15.0.*
+    - pyarrow
```

@jakirkham not sure if this is the right approach, but looks like CPU packages get pulled otherwise:
```suggestion
    - pyarrow
    - arrow-cpp-proc * cuda
```

---

Interesting. I would have thought that constraining this in `libcudf` would be sufficient. Maybe our `run_export` constraint in `libcudf` is not strict enough? If we do need this in `host`, we should add it to `run` as well.

---

It looks like just adding it to the `run` for `libcudf` seems to have resolved the issues so we can just go with that for now.

---

Ok we should keep an eye on this though. Locally built packages effectively live in a channel of highest priority. The behavior may be different when we have multiple packages in `rapidsai-nightly` to choose from.